app/src/main/java/com/github/gotify/messages/MessagesActivity.java

```diff
@@ -360,6 +363,54 @@ protected void onDestroy() {
         picasso.shutdown();
     }
 
+    private void scheduleDeletion(int position, Message message) {
```

Comment:
Do know where to put this:

* Delete a message
* Scroll a little bit to the bottom
* Undo Delete
* Try delete again
-> Delete doesn't work

 [gif with bugged delete](https://user-images.githubusercontent.com/14895212/74355543-ea628a80-4db4-11ea-9703-cea377839088.gif)

Reply:
Good catch. `messagesView.smoothScrollToPosition(insertPosition);` seems to cause it. If I remove that line I can't reproduce it. I didn't manage to find out what's wrong with it, so I'm just going to leave it out.